### PR TITLE
Enable ironic deployment against newton

### PR DESCRIPTION
**Pull request by mkam**
Enable ironic by overwritting what conf.d
files are copied for the aio scenario.

### CODE REVIEW

**Comment by mkam** [comment]
recheck_cit_trusty_mitaka
recheck_cit_trusty_newton
recheck_cit_xenial_newton

File: tests/aio-create.sh [hunk 1]
```diff
@@ -120,6 +138,7 @@ pushd /opt/openstack-ansible
 
   elif [ "${IRR_CONTEXT}" == "newton" ]; then
     git checkout "stable/newton"  # Branch checkout of Newton (Current Stable)
+    enable_ironic
```
Comment (cloudnull) [inline tests/aio-create.sh]: I think this should be added to newton+ for future testing and maintainability.

Comment (mkam) [reply tests/aio-create.sh]: Added it to ocata.

**Comment by mkam** [comment]
recheck_cit_trusty_newton

**Pull request merged**